packages/node/src/__tests__/http-client.integration.test.ts

```diff
@@ -20,7 +20,6 @@ const helpers = {
   ) => {
     expect(url).toBe('https://api.segment.io/v1/batch')
     expect(options.headers).toEqual({
-      Authorization: 'Basic Zm9vOg==',
```

Comment:
Since we're removing the encoded writeKey from the header, can we add a test or update a matcher to make sure writeKey is being added to events and/or the root batch?

Reply:
Nevermind - next file shows you are checking that (in http-integration.test.ts)